howdy/src/pam/main.cc

```diff
@@ -193,7 +195,16 @@ auto check_enabled(const INIReader &config) -> int {
  */
 auto identify(pam_handle_t *pamh, int flags, int argc, const char **argv,
               bool auth_tok) -> int {
-  INIReader config("/lib/security/howdy/config.ini");
+  if (!strlen(LIB_PATH) || LIB_PATH == NULL) {
+    syslog(LOG_ERR, "Unable to find Howdy's installation location");
+    return PAM_SYSTEM_ERR;
+  }
+
+  char CONFIG_PATH[strlen(LIB_PATH) + strlen(CONFIG_FILE) + 1]; // + 1 terminating \0
+  strcpy(CONFIG_PATH, LIB_PATH);
+  strcat(CONFIG_PATH, CONFIG_FILE);
```

Comment:
Why not using standard C++ way, e.g. converting to `std::string` and concatenating with `+` operator?

Reply:
`const char *const args[] = {PYTHON_EXECUTABLE, COMPARE_PATH, username, nullptr};` requires char arrays, so I preferred to write it in C-style, in order to avoid a string -> char* conversion

Reply:
But again, I can change that if you prefer